README.md

```diff
@@ -24,39 +24,40 @@
 An abstract language model of SystemVerilog (incl. Verilog) written in Python.
 
 ## Main Goals
-This package provides a unified abstract language model for SystemVerilog (incl.
-Verilog). Projects reading from source files can derive own classes and implement
-additional logic to create a concrete language model for their tools.
 
-Projects consuming pre-processed VHDL data (parsed, analyzed or elaborated) can
-build higher level features and services on such a model, while supporting multiple
-frontends.
+This package provides a unified abstract language model for SystemVerilog (incl. Verilog).
+Projects reading from source files can derive own classes and implement additional logic to create a concrete language
+model for their tools.
+
+Projects consuming pre-processed System Verilog data (parsed, analyzed or elaborated) can build higher level features
+and services on such a model, while supporting multiple frontends.
 
 
 ## Use Cases
-**pySystemVerilogModel Generators**
```

Comment:
In pyVHDLModel I used just **bold** and no headline level as the headline gets quite big.